### PR TITLE
Usage reporting enhacement

### PR DESCRIPTION
**Pull request by cegganesh84**
…ters

1. Update usage.proto
2. Add clusterDefinitionName as the blueprint name
3. This is used to track the number of datamart and dataengineering datahub clusters.
4. In future usage of different flavours of datalake clusters.

./gradlew clean build

### CODE REVIEW

File: usage-collection/src/main/proto/usage.proto [hunk 1]
```diff
@@ -38,6 +38,8 @@ message Event {
         AltusIamUserUpdated altusIamUserUpdated = 6;
         // An Altus user interactively logged in.
         AltusIamUserInteractiveLogin altusIamUserInteractiveLogin = 7;
+        // An Altus user interactively login failure.
```
Comment (bhavanki) [inline usage-collection/src/main/proto/usage.proto]: Nit: "An Altus user failed to interactively log in" is better grammatically.

Comment (cegganesh84) [reply usage-collection/src/main/proto/usage.proto]: Thanks Bill! This is a copy of the file from the thunderhead repo. I will fix it there next time I make an enhancement.

http://ci-cloudbreak.eng.hortonworks.com/job/cloudbreak-pull-request-builder-integration-test/4033/ -> Is it possible to look at what is the failure? I am getting 404. Thanks!

Comment (bhavanki) [reply usage-collection/src/main/proto/usage.proto]: Unfortunately, that build aged out of the Jenkins server already. It doesn't take very long. You'll have to update your branch to retrigger the job (a force push should do it, if necessary).

**Comment by cegganesh84** [comment]
recheck

**Comment by cegganesh84** [comment]
recheck

**Comment by keyki** [comment]
Branch should be 2.16 instead of 1.16

**Comment by cegganesh84** [comment]
@keyki thank you!

**Comment by cegganesh84** [comment]
@keyki Could you please merge this?

**Comment by cegganesh84** [comment]
@keyki thanks again!

**Pull request merged**